src/MSBuild/XMake.cs

```diff
@@ -2092,10 +2092,13 @@ out enableProfiler
                         Console.WriteLine(ResourceUtilities.FormatResourceStringStripCodeAndKeyword("PickedUpSwitchesFromAutoResponse", autoResponseFileName));
                     }
 
-                    if (verbosity == LoggerVerbosity.Diagnostic)
+                    if (verbosity >= LoggerVerbosity.Detailed)
                     {
+                        string currentWorikingDirectory = Directory.GetCurrentDirectory();
                         string equivalentCommandLine = commandLineSwitches.GetEquivalentCommandLineExceptProjectFile();
```

Comment:
It might be useful here to also log the path to dotnet.exe
```C#
#if RUNTIME_TYPE_NETCORE
    var hostProcess = Process.GetCurrentProcess()?.MainModule?.FileName;
    Console.WriteLine($"Dotnet host path: {hostProcess}");
#endif
```